### ISSUE TITLE
ansible playbook fails to start

### ISSUE THREAD

**Issue by cmoulliard** [RedHat-EMEA-SSA-Team/hetzner-ocp4#73]
## Issue

The command `ansible-playbook ./ansible/setup.yml` reports the following error 

```
[root@ocp4 hetzner-ocp4]# ansible-playbook ./ansible/setup.yml
 [WARNING]: Could not match supplied host pattern, ignoring: all

 [WARNING]: provided hosts list is empty, only localhost is available

ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '/root/temp/hetzner-ocp4/ansible/roles/openshift-4-loadbalancer/tasks/create.yml': line 25, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Collect services facts
  ^ here


The error appears to have been in '/root/temp/hetzner-ocp4/ansible/roles/openshift-4-loadbalancer/tasks/create.yml': line 25, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Collect services facts
  ^ here

exception type: <class 'ansible.errors.AnsibleParserError'>
exception: no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '/root/temp/hetzner-ocp4/ansible/roles/openshift-4-loadbalancer/tasks/create.yml': line 25, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Collect services facts
  ^ here
```

### Additional info

OS: Centos7
Ansible version : 2.4.2.0
cluster.yml
```
---
cluster_name: ocp4
public_domain: example.com
dns_provider: [route53|cloudflare|gcp]
letsencrypt_account_email: cmoulliard@redhat.com
# Depending on the dns provider:
# CloudFlare
cloudflare_account_email: john@example.com
cloudflare_account_api_token: 9348234sdsd894.....
cloudflare_zone: example.com
# Route53
aws_access_key: key
aws_secret_key: secret
aws_zone: example.com
# GCP
gcp_project: project-name
gcp_managed_zone_name: 'zone-name'
gcp_managed_zone_domain: 'example.com.'
gcp_serviceaccount_file: ../gcp_service_account.json

auth_htpasswd:
  - admin:$ttttttttt//
  - local:$ttttttttt//

storage_nfs: false # Default is false

auth_redhatsso:
  client_id: "xxxxx.apps.googleusercontent.com"
  client_secret: "xxxxxxx"

cluster_role_bindings:
  - cluster_role: sudoers
    name: login@redhat.com
  - cluster_role: cluster-admin
    name: admin


image_pull_secret: |-
  ttttttttt
```

**Comment by cmoulliard** [comment]
Version of ansible 2.4 is not supported for your playbooks.

Could you please mention what is the minimal version to be used please ?

I'm doing a test with `2.9` which is working ;-)
```
ansible-playbook ./ansible/setup.yml
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'


PLAY [localhost] ******************************************************************************************************************************************************************************

TASK [Gathering Facts] ************************************************************************************************************************************************************************
Friday 20 December 2019  18:41:50 +0100 (0:00:00.027)       0:00:00.027 *******
ok: [localhost]

TASK [openshift-4-cluster : Installing KVM Packages (OS Release == 7).] ***********************************************************************************************************************
Friday 20 December 2019  18:41:51 +0100 (0:00:00.952)       0:00:00.979 *******
changed: [localhost]
...
```

**Comment by rbo** [comment]
Good point, minimum ansible version is 2.8

**Comment by rbo** [comment]
Will be merged with PR #75
